### PR TITLE
TM-5600-X-AIM-Tabs

### PR DESCRIPTION
**Pull request by cagbakwu**
[Ticket](https://metaphase.atlassian.net/browse/TM-5600)
[Video](https://github.com/MetaPhase-Consulting/State-TalentMAP/assets/135657303/92df4b2f-c5a9-478e-9b68-e1c8d3c24524)

### CODE REVIEW

File: src/Components/Agenda/AgendaItemResearchPane/FrequentPositions/FrequentPositions.jsx [hunk 1]
```diff
@@ -49,22 +49,25 @@ const FrequentPositions = (props) => {
...
+            value={term}
+            id={'frequent-positions-search'}
+            labelSrOnly
+            placeholder="Search for Frequent Positions"
+            inputProps={{
+              autoComplete: 'off',
+            }}
+          />
+        </FieldSet>
+        <FA name={term && 'close'} onClick={() => setTerm('')} />
```
Comment (cagbakwu) [inline src/Components/Agenda/AgendaItemResearchPane/FrequentPositions/FrequentPositions.jsx:69]: Added the 'Close' icon here for this input box.

File: src/Components/Agenda/AgendaItemResearchPane/GsaLocations/GsaLocations.jsx [hunk 2]
```diff
@@ -74,7 +74,9 @@ const GsaLocations = ({ setLocation, activeAIL }) => {
             placeholder="Search city"
             onChange={(e) => setCity(e.target.value)}
             onKeyUp={(e) => { if (ifEnter(e)) setIsSearching(!isSearching); }}
+            value={city}
```
Comment (cagbakwu) [inline src/Components/Agenda/AgendaItemResearchPane/GsaLocations/GsaLocations.jsx:77]: added the values for each of these inputs, along with the close icons and conditions to appear and clear.

File: src/Components/Agenda/AgendaItemResearchPane/RemarksGlossary/RemarksGlossary.jsx [hunk 3]
```diff
@@ -134,16 +134,19 @@ const RemarksGlossary = ({ isReadOnly, remarks, remarkCategories,
...
+          changeText={e => { search(e); setTerm(e); }}
+          value={term}
+          labelSrOnly
+          placeholder="Search for Remarks"
+          id="remarks-search"
+          inputProps={{
+            autoComplete: 'off',
+          }}
+        />
+        <FA className="close-fa" name={term && 'close'} onClick={() => setTerm('')} />
```
Comment (cagbakwu) [inline src/Components/Agenda/AgendaItemResearchPane/RemarksGlossary/RemarksGlossary.jsx:148]: Added the 'Close' icon here for this input box.

**Comment by M00staff** [comment]
This seems off...

image

**Comment by cagbakwu** [comment]
@M00staff whoa what happed to my CSS?! let me check the local and see if something changed.

**Review by M00staff — APPROVED** [review]
👍

**Pull request merged**